segmentation/utils/visualize_test.py

```diff
@@ -10,3 +16,25 @@ def test_visualize_z_scores():
     # Assign random phenotype titles
     pheno_titles = [chr(i) for i in range(ord('a'), ord('z') + 1)]
     plot = visualize.visualize_z_scores(z, pheno_titles)
+
+
+def test_get_sort_data():
+    rand_type = random.choices(string.ascii_lowercase, k=100)
+    ids = random.choices(range(1, 10), k=100)
+    random_data = pd.DataFrame.from_dict({"PatientID": ids, "cell_type": rand_type})
+    sorted_data = visualize.get_sorted_data(random_data, "PatientID", "cell_type")
+
+    row_sums = [row.sum() for index, row in sorted_data.iterrows()]
+    assert sorted(row_sums)
```

Comment:
This assertion only tests for the existence of `sorted(row_sums)` aka that it's not `None`, so it will never fail. If you want to assert that `row_sums` is sorted properly, you should test for equality:

`row_sums == sorted(row_sums)`

By the way, I ran the function and row_sums actually looks like it's sorted in reverse order by default. In that case, you can do `assert list(reversed(row_sums)) == sorted(row_sums)`.